Regression in health checking under 4.18.0

```
hyperbahn:production:HOST-hyperbahn-X health check
Details
Result: Failed (exit code 8)
Command: tcurl -p localhost:21300 autobahn health_v1
Stderr: 
assert.js:92
  throw new assert.AssertionError({
        ^
AssertionError: must send a body
    at TChannelJSON.send (.../tcurl/node_modules/tchannel/as/json.js:97:5)
    at TCurl.asJSON (.../tcurl/index.js:496:12)
    at TCurl.send (.../tcurl/index.js:385:14)
    at onReady (.../tcurl/index.js:403:14)
    at onIdentified (.../tcurl/index.js:362:9)
    at finish (.../tcurl/node_modules/tchannel/peer.js:452:9)
    at Array.onIdentified [as 1] (.../tcurl/node_modules/tchannel/peer.js:443:9)
    at DefinedEvent.emit (.../tcurl/node_modules/tchannel/lib/event_emitter.js:90:25)
    at TChannelConnection.onOutIdentified (.../tcurl/node_modules/tchannel/connection.js:541:26)
    at DefinedEvent.onOutIdentified [as listener] (.../tcurl/node_modules/tchannel/connection.js:520:14)
```

--- a/index.js
+++ b/index.js
@@ -494,7 +494,7 @@ TCurl.prototype.asJSON = function asJSON(opts, request, delegate, done) {
     var self = this;
     var sender = new TChannelAsJSON();
     sender.send(request, opts.endpoint, opts.jsonHead,
-        opts.jsonBody, onResponse);
+        opts.jsonBody || {}, onResponse);
 
     function onResponse(err, res, arg2, arg3) {
         done();

--- a/test/as-json.js
+++ b/test/as-json.js
@@ -105,6 +105,68 @@ test('getting an ok response', function t(assert) {
     }
 });
 
+test('getting an ok response with implied body', function t(assert) {
+    var server = new TChannel({
+        serviceName: 'server'
+    });
+    var opts = {
+        isOptions: true
+    };
+
+    var hostname = '127.0.0.1';
+    var port;
+    var endpoint = 'echo';
+    var head = {some: 'echo-head'};
+    var body = {};
+    var serviceName = 'server';
+
+    var tchannelJSON = TChannelJSON();
+    tchannelJSON.register(server, endpoint, opts, echo);
+
+    function onServerListen() {
+        port = server.address().port;
+        onListening();
+    }
+
+    server.listen(0, hostname, onServerListen);
+
+    function onListening() {
+        var cmd = [
+            '-p', hostname + ':' + port,
+            serviceName,
+            endpoint,
+            '-2', JSON.stringify(head),
+            '-j'
+        ];
+
+        tcurl.exec(cmd, {
+            error: function error(err) {
+                assert.ifError(err);
+            },
+            response: function response(res) {
+                assert.deepEqual(res, {
+                    ok: true,
+                    head: null,
+                    body: {
+                        opts: {
+                            isOptions: true
+                        },
+                        head: head,
+                        body: body,
+                        serviceName: serviceName
+                    },
+                    headers: {
+                        'as': 'json'
+                    }
+                });
+            },
+            exit: function exit() {
+                server.close();
+                assert.end();
+            }
+        });
+    }
+});
 function slowEcho(opts, req, head, body, cb) {
     timers.setTimeout(function slowReturn() {
         cb(null, {

